### DISCUSSION TITLE
[enhancement] [feature request] Virtual Scroll

### DISCUSSION THREAD

**Discussion by em2046** [element-plus/element-plus#939] [Ideas]
el-table
el-select
el-tree
Support virtual scroll get better performance

Please refer to [AntDesign](https://ant.design/components/table-cn/#components-table-demo-virtual-list)

**Comment by zazzaz** [comment]
#790

**Comment by jw-foss** [comment]
Thank you for bring this up, it might take some time for us to implement them with virtual list. But it's on our road map, we hope that we could roll it out soon

**Comment by zychome** [comment]
期待...

**Comment by yudan215** [comment]
希望可以增加类似select无限加载+虚拟滚动功能·

**Comment by zychome** [comment]
http://www.quasarchs.com/vue-components/virtual-scroll

element-plus团队  可以借鉴一下quasar的虚拟滚动。加油！！！

**Comment by wfk007** [comment]
期待
